pkgs/development/python-modules/langchain-core/default.nix

```diff
@@ -89,6 +86,12 @@ buildPythonPackage rec {
     '';
   };
 
+  disabledTests = lib.optionals stdenv.isDarwin [
+    # Langchain-core the following tests due to the test comparing execution time with magic values.
+    "test_queue_for_streaming_via_sync_call"
+    "test_same_event_loop"
+  ];
+
```

Comment:
I would have this snippet before `passthru` but it's not a big deal, so feel free to ignore.

Reply:
As you don't really mind, I can take care of that during the next PR for adding `langchain` stuff.